### PR TITLE
Remove Image in Example of <filter> svg ref

### PR DESCRIPTION
**Pull request by digi-booster**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

1. Remove screenshot image of live example
2. increase height to remove vertical scrolling

> Anything else that could help us review it
[99.3% global support of svg filter](https://caniuse.com/svg-filters)

### CODE REVIEW

File: files/en-us/web/svg/element/filter/index.md [hunk 1]
```diff
@@ -45,21 +45,21 @@ This element implements the {{domxref("SVGFilterElement")}} interface.
...
+  <circle cx="60" cy="60" r="50" fill="green"/>
+  
+  <circle cx="170" cy="60" r="50" fill="green" filter="url(#blurMe)"/>
 </svg>
 ```
 
 #### Result
 
-{{EmbedLiveSample("Example",232,124,"/files/4227/feGaussianBlur.png")}}
+{{EmbedLiveSample("Example",235,135)}}
```
Comment (digi-booster) [inline files/en-us/web/svg/element/filter/index.md:62]:
```suggestion
{{EmbedLiveSample("Example",235,140)}}
```

File: files/en-us/web/svg/element/filter/index.md [hunk 2]
```diff
@@ -45,21 +45,21 @@ This element implements the {{domxref("SVGFilterElement")}} interface.
...
+  <circle cx="60" cy="60" r="50" fill="green"/>
+  
+  <circle cx="170" cy="60" r="50" fill="green" filter="url(#blurMe)"/>
 </svg>
 ```
 
 #### Result
 
-{{EmbedLiveSample("Example",232,124,"/files/4227/feGaussianBlur.png")}}
+{{EmbedLiveSample("Example",235,140)}}
```
Comment (digi-booster) [inline files/en-us/web/svg/element/filter/index.md:62]:
```suggestion
{{EmbedLiveSample("Example",235,145)}}
```

File: files/en-us/web/svg/element/filter/index.md [hunk 3]
```diff
@@ -45,21 +45,21 @@ This element implements the {{domxref("SVGFilterElement")}} interface.
...
+  <circle cx="60" cy="60" r="50" fill="green"/>
+  
+  <circle cx="170" cy="60" r="50" fill="green" filter="url(#blurMe)"/>
 </svg>
 ```
 
 #### Result
 
-{{EmbedLiveSample("Example",232,124,"/files/4227/feGaussianBlur.png")}}
+{{EmbedLiveSample("Example",235,145)}}
```
Comment (digi-booster) [inline files/en-us/web/svg/element/filter/index.md:62]:
```suggestion
{{EmbedLiveSample("Example",235,150)}}
```

File: files/en-us/web/svg/element/filter/index.md [hunk 4]
```diff
@@ -45,21 +45,21 @@ This element implements the {{domxref("SVGFilterElement")}} interface.
...
-<svg width="230" height="120" xmlns="http://www.w3.org/2000/svg">
- <filter id="blurMe">
-   <feGaussianBlur stdDeviation="5"/>
- </filter>
-
- <circle cx="60" cy="60" r="50" fill="green" />
-
- <circle cx="170" cy="60" r="50" fill="green"
-          filter="url(#blurMe)" />
+<svg height="120" width="230" xmlns="http://www.w3.org/2000/svg">
```
Comment (digi-booster) [inline files/en-us/web/svg/element/filter/index.md:48]:
```suggestion
<svg width="230" height="120" xmlns="http://www.w3.org/2000/svg">
```

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Thanks @digi-booster 👍

**Pull request merged**